scripts/sct_convert.py

```diff
@@ -54,21 +57,19 @@ def get_parser():
 
 # conversion
 # ==========================================================================================
-def convert(fname_in, fname_out, squeeze_data=True, type=None, verbose=1):
+def convert(fname_in, fname_out, squeeze_data=True, dtype=None, verbose=1):
     """
     Convert data
     :return True/False
     """
-    from msct_image import Image
-    from sct_utils import printv
-    printv('sct_convert -i ' + fname_in + ' -o ' + fname_out, verbose, 'code')
-    # Open file
-    im = Image(fname_in)
-    # Save file
-    im.setFileName(fname_out)
-    if type is not None:
-        im.changeType(type=type)
-    im.save(squeeze_data=squeeze_data)
+    import msct_image
+    sct.printv('sct_convert -i ' + fname_in + ' -o ' + fname_out, verbose, 'code')
+    im = msct_image.Image(fname_in)
+    if squeeze_data:
+        im.data = np.squeeze(im.data)
```

Comment:
i'm still puzzled as to why in this function `squeeze` is on by default. We probably had some issues in the past but i don't remember what. Maybe looking at old closed issues.

Reply:
We don't remember because no reference to issues or useful comment was left in the code... anyway, we'll see what happens when running tests.